src/natcap/invest/urban_nature_access.py

```diff
@@ -0,0 +1,315 @@
+import os
+import logging
+import shutil
+import tempfile
+
+import pygeoprocessing
+import taskgraph
+from osgeo import gdal
+from osgeo import osr
+import numpy
+
+from . import validation
+from . import spec_utils
+from . import utils
+from .spec_utils import u
+from natcap.invest import MODEL_METADATA
+
+LOGGER = logging.getLogger(__name__)
+UINT32_NODATA = int(numpy.iinfo(numpy.uint32).max)
+FLOAT32_NODATA = float(numpy.finfo(numpy.float32).max)
+ARGS_SPEC = {
+    'model_name': MODEL_METADATA['urban_nature_access'].model_title,
+    'pyname': MODEL_METADATA['urban_nature_access'].pyname,
+    'userguide_html': MODEL_METADATA['urban_nature_access'].userguide,
+    'args_with_spatial_overlap': {
+        'spatial_keys': [],
+        'different_projections_ok': True,
+    },
+    'args': {
+        'workspace_dir': spec_utils.WORKSPACE,
+        'results_suffix': spec_utils.SUFFIX,
+        'n_workers': spec_utils.N_WORKERS,
+        'lulc_raster_path': {
+            **spec_utils.LULC,
+            'projected': True,
+            'projection_units': u.meter,
+            'about': (
+                "A map of LULC codes. "
+                "All values in this raster must have corresponding entries "
+                "in the LULC attribute table."),
+        },
+        'lulc_attribute_table': {
+            'name': 'LULC attribute table',
+            'type': 'csv',
+            'columns': {
+                'lucode': {
+                    'type': 'integer',
+                    'about': (
+                        "LULC code.  Every value in the LULC map must have a "
+                        "corresponding entry in this column."),
+                },
+                'greenspace': {
+                    'type': 'number',
+                    'units': u.none,
+                    'about': (
+                        "1 if this landcover code represents greenspace, 0 "
+                        "if not."
+                    ),
+                }
+            },
+            'about': (
+                "A table identifying which LULC codes represent greenspace."
+            ),
+        },
+        'population_raster_path': {
+            'type': 'raster',
+            'name': 'population raster',
+            'bands': {
+                1: {'type': 'number', 'units': u.none}
+            },
+            'projected': True,
+            'about': (
+                "A raster representing the number of people who live in each "
+                "pixel. This raster must be linearly projected."
```

Comment:
Not only linearly projected, but meters specifically I think.

Reply:
```suggestion
                "pixel. This raster must be linearly projected in meters."
```

Reply:
Yeah, I kind of went back and forth on this idea, but after digging into the `GetLinearUnits` source code I think you're right.  I really don't like that `GetLinearUnits` will silently fall back to "oh well, let's just assume meters" and that will for sure cause us more headaches down the line than just requiring a linear projection in meters for the LULC and the population raster.  So that's been added in 32834c43